src/Controls/tests/Core.UnitTests/PointerGestureRecognizerTests.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Linq;
+using Microsoft.Maui.Controls.Internals;
+using Microsoft.Maui.Graphics;
+using Xunit;
+
+namespace Microsoft.Maui.Controls.Core.UnitTests
+{
+	public class PointerGestureRecognizerTests : BaseTestFixture
+	{
+		[Fact]
+		public void Constructor()
+		{
+			var pointer = new PointerGestureRecognizer();
+		}
+
+		[Fact]
+		public void PointerOverVSMGestureOnlyPresentWithVSM()
```

Comment:
can't actually remember the acronym rule myself, but can we double check to see which is relevant here?

```suggestion
		public void PointerOverVsmGestureIsOnlyPresentWithVsm()
```